doc/en/user/source/extensions/vectortiles/tutorial.rst

```diff
@@ -80,18 +80,17 @@ For this tutorial, we'll be using the layer ``opengeo:countries`` to show off th
 
 Our layer is now ready to be served.
 
-Create OpenLayers application
------------------------------
+Create OpenLayers application - TMS Vector Tiles
+------------------------------------------------
 
-#. Create a ``www/vectortiles`` directory inside your GeoServer Data Directory.
+#. Create a ``www/tms-vectortiles`` directory inside your GeoServer Data Directory.
 
-#. Download the `latest version of OpenLayers <http://openlayers.org/download/>`_.
+#. Download the `latest version of OpenLayers <http://openlayers.org/download/>. Download the v<ol-version>-package.zip file`_.
```

Comment:
The docs build fails with:

```
07:36:54,213 [INFO]      [exec] /home/runner/work/geoserver/geoserver/doc/en/user/source/extensions/vectortiles/tutorial.rst:88: ERROR: Unknown target name: "latest version of openlayers <http://openlayers.org/download/>. download the v<ol-version>-package.zip file"."
```

The link seems to be missing a small bit at  the end, hopefully this is going to fix it.

```suggestion
#. Download the `latest version of OpenLayers <http://openlayers.org/download/>`_. Download the v<ol-version>-package.zip file`_.
```

Reply:
Thanks